commitizen/git.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, rev, title, body=""):
 
     @property
     def message(self):
-        return f"{self.title}\n\n{self.body}"
+        message = self.title
+        if self.body:
+            message = message + f"\n\n{self.body}"
+        return message
```

Comment:
How about using `f"{self.title}\n\n{self.body}".strip()` instead?

Reply:
Yep
That will be better